### PR TITLE
Applied Color Grading and Adde Patient Testimonial Text

### LINKED ISSUE #827: 🐞[Bug]: Color Grading Issues and add Patient Testimonial Text
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hey @varshith257 

heading and paragraph for some sections is not visible due to it's color such as Appointment section, Patient portal section, Contact Us section and services section

image

image

image

image

image

image

I would like to add review text for testimonial section

image

Could you please assign this issue under GSSOC24?

### Expected behavior

N/A

### Add ScreenShots

_No response_

### What browsers are you seeing the problem on?

_No response_

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by RadhikaMalpani1702**
## Description
Hey @varshith257 

I have Applied Color Grading and added Patient Testimonial Text.

image
image
image
image
image
image

## Related Issues

- Closes #827 

## Type of PR

- [X] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -504,7 +504,7 @@ <h2>Patient Portal</h2>
     <section id="appointment" class="appointment section-bg">
       <div class="container" style="border-radius: 10px;">
         <div class="section-title section-black-text">
-          <h2>Make an Appointment</h2>
+          <h2 style="color: white;">Make an Appointment</h2>
```
Comment (varshith257) [inline index.html:507]:
```suggestion
          <h2 style="color: white;">Make an Appointment</h2>
L
```

Can this be moved to Css? Hardcoded color tag in index.html couldn't be good

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Comment by RadhikaMalpani1702** [comment]
@varshith257 I have done the changes please merge this PR

**Comment by RadhikaMalpani1702** [comment]
@varshith257 Please merge this PR

**Comment by varshith257** [comment]
@Anjaliavv51 @Pruthaa17 @aditya-bhaumik Have a look on this too

**Comment by aditya-bhaumik** [comment]
@varshith257 hi i think this pull request can be merged

**Review by aditya-bhaumik — APPROVED** [review]

**Comment by Anjaliavv51** [comment]
@varshith257 this PR can be merged , all changes are done.

**Review by varshith257 — APPROVED** [review]

**Pull request merged**